source/agora/common/Types.d

```diff
@@ -58,3 +58,6 @@ public struct QuorumConfig
     /// List of any sub-quorums
     public QuorumConfig[] quorums;
 }
+
+/// Delegate to get the number of active validator at the block height
+public alias ValidatorCountGetter = scope ulong delegate (ulong block_height) @safe nothrow;
```

Comment:
I added a delegate.
I couldn't find a good location. So I added it here.

Reply:
I don't see why this is necessary at all. Just pass it as a count?

Reply:
You're right. That's a much better way.

Reply:
@AndrejMitrovic: I fixed it. thank you for review.